schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+  id     INT GENERATED ALWAYS AS IDENTITY,
+  name    varchar,
+  date_of_birth  date,
+  escape_attempts int,
+  neutered  bit,
+  weight_kg real
 );
```

Comment:
[OPTIONAL] I suggest to mark the id column as primary key `PRIMARY KEY (id)` 💯

Reply:
@Bondok6 I forgot it.   I already added it. 
Thanks for let me know  !
![image](https://user-images.githubusercontent.com/17918694/196314624-9eca34ef-e496-47a0-ab4c-bd163be9323e.png)